src/app/container/add-tag/add-tag.component.ts

```diff
@@ -139,7 +139,7 @@ export class AddTagComponent extends Base implements OnInit, AfterViewChecked {
   addTag() {
     this.alertService.start('Adding tag');
     this.containertagsService.addTags(this.tool.id, [this.unsavedVersion]).subscribe((tags: Tag[]) => {
-      this.tool.tags = tags;
+      this.tool.workflowVersions = tags;
```

Comment:
A bit weird it is called workflowVersions for tools. Should it just be versions?

Reply:
I think this is swagger codegen from the webservice. 
For background, workflows currently have workflowVersions, tools have tags.

There are few options to unify things:
1) Standardize on tags and break users of workflows like older CLIs and DNAstack)
2) Standardize on workflowVersions and break users of tools
3) Standardize on versions and break both

The proposal in these two linked PRs is to do 1 and send back the old json structure with tags when we detect a CLI with an old user agent